climakitae/explore/warming.py

```diff
@@ -131,9 +134,9 @@ def calculate(self):
             gwl_snapshots=self.gwl_snapshots,
             wl_params=self.wl_params,
             cmap=self.cmap,
-            warming_levels=self.warming_levels,
+            warming_levels=self.wl_params.warming_levels,
         )
-        self.wl_viz.compute_stamps()
+        # self.wl_viz.compute_stamps()
```

Comment:
Temporary fix while WL is being upgraded in other PRs. This will be uncommented later, and will not have an effect on this PR once those other ones are merged in.

Reply:
In its current state, this will actually break `warming_levels.ipynb` and `warming_level_approach.ipynb`.